src/GAP_pkg.jl

```diff
@@ -96,9 +102,14 @@ function setup_overrides()
 
     # GAP package "singular" uses `Singular` executable from Singular_jll
     singular_binpath = @generate_wrappers(Singular_jll)
-    Globals.sing_exec = GapObj(joinpath(singular_binpath, "Singular"))
+    GAP.Globals.sing_exec = GapObj(joinpath(singular_binpath, "Singular"))
     d = GAP.Globals.Directory(GapObj(singular_binpath))
     GAP.Globals.Add(GAP.Globals.GAPInfo.DirectoriesPrograms, d)
+
+    # GAP package "polymaking" uses the 'polymake' Perl script from polymake_jll
+    polymake_command = GapObj(generate_polymake_wrapper())
+    GAP.Globals.POLYMAKE_COMMAND = polymake_command
+    GAP.Globals.POLYMAKE_PATH = polymake_command
```

Comment:
```suggestion
    GAP.Globals.POLYMAKE_PATH = polymake_command * " "
```
hap puts a trailing space to the path itself (cf https://github.com/gap-packages/hap/blob/7def4be17667cd3b9e7a080ac96697eda5c37545/lib/externalSoftware.gap#L17). This seems then to be required such that concatenating the POLYMAKE_PATH and the input file produces a valid call (cf https://github.com/gap-packages/hap/blob/7def4be17667cd3b9e7a080ac96697eda5c37545/lib/Polymake/orbitPoly.gi#L57).
This could be another point for your discussion with Graham Ellis